### PR TITLE
Remove namespace

### PR DESCRIPTION
**Pull request by foxish**
Fixes https://github.com/apache-spark-on-k8s/kubernetes-HDFS/issues/6

### CODE REVIEW

File: charts/hdfs-datanode-k8s/README.md [hunk 1]
```diff
@@ -29,14 +29,14 @@ HDFS `datanodes` running inside a kubernetes cluster. See the other chart for
...
+  $ helm install -n my-hdfs-datanode \
       --set clusterDnsIP=YOUR-KUBE-DNS-IP hdfs-datanode-k8s
   ```
 
   5. Confirm the daemons are launched.
 
   ```
-  $ kubectl get pods --all-namespaces | grep hdfs-datanode-
+  $ kubectl get pods | grep hdfs-datanode-
   kube-system   hdfs-datanode-ajdcz 1/1 Running 0 7m
```
Comment (kimoonkim) [inline charts/hdfs-datanode-k8s/README.md]: Remove "kube-system" in this example output?

Comment (foxish) [inline charts/hdfs-datanode-k8s/README.md]: Done. Thanks!

**Comment by foxish** [comment]
@kimoonkim ok to merge this?

File: charts/hdfs-datanode-k8s/README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ HDFS `datanodes` running inside a kubernetes cluster. See the other chart for
      It will be supplied below as the `clusterDnsIP` parameter.
 
   ```
-  $ kubectl get svc --all-namespaces | grep kube-dns
```
Comment (kimoonkim) [inline charts/hdfs-datanode-k8s/README.md]: Ah. I think we should revert to the previous line since kube-dns is not in the `default` namespace.

**Comment by kimoonkim** [comment]
@foxish This change is pretty much ready. Any chance we can fix the README.md and merge soon? FYI, I don't think I have the push privilege for this repo in case you were waiting for me to do the merge.

**Comment by foxish** [comment]
Done, merging. Will add the push privilege for you in this repo. Thanks!

**Comment by kimoonkim** [comment]
Cool. Thanks for taking care of this! And happy to have the push privilege :)

**Pull request merged**